.circleci/config.yml

```diff
@@ -399,16 +254,53 @@ workflows:
       - test-SmartSync
       - test-SalesforceSwiftSDK
 
-  # Cron are on a timezone 8 hours ahead of PST
-  # Nightly run is set for midnight Monday - Friday
-  nightly-test-ios:
+  # Cron is UTC timezone, which is 7 hours ahead of PST.
+  # Run tests at 1am.
+  nightly-test-ios11:
+    triggers:
+      - schedule:
+          cron: "00 08 * * 2,3,4,5,6"
+          filters:
+            branches:
+              only:
+                - dev
+    jobs:
+      - test-SalesforceAnalytics:
+            context: nightly-test
```

Comment:
Shouldn't this be `nightly-test-ios11` (same as line 259 above)?

Reply:
Nope, 259 is the workflow name. This context name should be nightly-test, it's actually the same one you already added for Android the other day. Since I have fastlane default to latest iOS version/device no extra work is needed there.

Then for the iOS 10 tests we will make a second one, nightly-test-ios10, that has the same nightly env var and an env var to specify iOS 10.

Reply:
Cool.